kalite/version.py

```diff
@@ -5,10 +5,15 @@
 # THIS IS USED BY settings.py.  NEVER import settings.py here; hard-codes only!
 # Must be PEP 440 compliant: https://www.python.org/dev/peps/pep-0440/
 # Must also be of the form N.N.N for internal use, where N is a non-negative integer
-MAJOR_VERSION = "0"
-MINOR_VERSION = "14"
-PATCH_VERSION = "0"
-VERSION = "%s.%s.%s" % (MAJOR_VERSION, MINOR_VERSION, PATCH_VERSION)
+
+from django.utils.version import get_version
+
+VERSION_TUPLE = (0, 14, 0, "alpha", 0)
+
+MAJOR_VERSION, MINOR_VERSION, PATCH_VERSION = map(str, VERSION_TUPLE[:3])
+
+VERSION = get_version(VERSION_TUPLE)
```

Comment:
So it will comply with the warning if it's a `"final"` release. Perhaps @jamalex can comment on whether that's acceptable? Unfortunately I can't supply an example of what a non-compliant version number will break.